src/Microsoft.Azure.Batch.SoftwareEntitlement.Common/Microsoft.Azure.Batch.SoftwareEntitlement.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.5</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
I think you'll want to remove these two `PropertyGroups` too (they define `NETSTANDARD1_5` which is out of sync with current version, but the rest of the values look like defaults so might as well remove the whole thing).